acra/configuring-maintaining/tls/_index.md

```diff
@@ -2,3 +2,146 @@
+
+* `--acraserver_tls_transport_enable={true|false}`
+
+  Enable TLS to encrypt transport between AcraConnector and AcraServer.
+  Default is `false` which means "use SecureSession instead".
+
+## AcraServer specific flags
+
+* `--acraconnector_tls_transport_enable={true|false}`
+
+  Enable TLS to encrypt transport between AcraConnector and AcraServer.
+  Default is `false` which means "use SecureSession instead".
+
+* `--tls_client_auth=<mode>`
+
+  Set authentication mode that will be used in TLS connection with AcraConnector.
+  Possible values are the same as for `--tls_auth`.
+  Default is `-1` which means "take value of `--tls_auth`".
+
+  Overrides the `--tls_auth` setting.
+
+* `--tls_client_key=<filename>`
+
+  Path to private key of the TLS certificate presented to AcraConnectors (see `--tls_client_cert`).
+  Empty by default.
+
+* `--tls_client_cert=<filename>`
+
+  Path to server TLS certificate presented to AcraConnectors (overrides `--tls_cert`).
+  Empty by default.
+
+* `--tls_client_ca=<filename>`
+
+  Path to additional CA certificate for AcraConnector certificate validation (setup if AcraConnector certificate CA is different from database certificate CA).
+  Empty by default.
+
+* `--tls_client_id_from_cert={true|false}`
+
+  Extract clientID from TLS certificate.
+  Take TLS certificate from AcraConnector's connection if `--acraconnector_tls_transport_enable` is `true`;
+  otherwise take TLS certificate from application's connection if `--acraconnector_transport_encryption_disable` is `true`.
+  Can't be used with `--tls_client_auth=0` or `--tls_auth=0`.
+  Default is `false`.
+
+* `--tls_database_auth=<mode>`
+
+  Set authentication mode that will be used in TLS connection with database.
+  Possible values are the same as for `--tls_auth`.
+  Overrides the `--tls_auth` setting.
+  Default is `-1` which means "take value of `--tls_auth`".
+
+* `--tls_database_key=<filename>`
+
+  Path to private key of the TLS certificate used to connect to database (see `--tls_database_cert`).
+  Empty by default.
+
+* `--tls_database_cert=<filename>`
+
+  Path to client TLS certificate shown to database during TLS handshake (overrides `--tls_cert`).
+  Empty by default.
+
+* `--tls_database_ca=<filename>`
+
+  Path to additional CA certificate for database certificate validation
+  (setup if database certificate CA is different from AcraConnector certificate CA).
+  Empty by default.
+
+* `--tls_database_sni=<SNI>`
+
+  Expected Server Name (SNI) from database.
+  Empty by default which means "don't check the SNI sent by database".
+
+* `--tls_db_sni=<SNI>`
+
+  Expected Server Name (SNI) from database (deprecated, use `--tls_database_sni` instead).
+
+## Other TLS flags
+
+* `--tls_identifier_extractor_type=<type>`
+
+  Decide which field of TLS certificate to use as ClientID.
+
+  * `distinguished_name` — (default) certificate Distinguished Name (DN)
+  * `serial_number` — certificate serial number
+
+  {{< hint warning >}}
```

Comment:
I'm not sure it it is rendered correctly. At least on GitHub display mode

Reply:
Don't worry, it's kind of extension, renders correctly with `hugo`, but not by GitHub's markdown renderer.